ext/libstrawberry-tagreader/tagreadertaglib.cpp

```diff
@@ -702,6 +579,138 @@ bool TagReaderTagLib::ReadFile(const QString &filename, spb::tagreader::SongMeta
 
 }
 
+void TagReaderTagLib::ParseID3v2Tag(TagLib::ID3v2::Tag *tag, QString *disc, QString *compilation, spb::tagreader::SongMetadata *song) const {
```

Comment:
Please add a newline after { for consistency with the rest of the code. And this function should be moved below TStringToStdString.

Reply:
Changed with the new commit